app/components/primer/beta/relative_time.rb

```diff
@@ -12,6 +12,9 @@ class RelativeTime < Primer::Component
       FORMAT_DEFAULT = :auto
       FORMAT_OPTIONS = [FORMAT_DEFAULT, :micro, :elapsed].freeze
 
+      FORMAT_STYLE_DEFAULT = nil
+      FORMAT_STYLE_OPTIONS = [FORMAT_STYLE_DEFAULT, :long, :short, :narrow].freeze
```

Comment:
https://github.com/github/relative-time-element#attributes

> Property Name | Attribute Name | Possible Values | Default Value
> -- | -- | -- | --
> formatStyle | format-style | 'long'\|'short'\|'narrow' | *
>
> *: If unspecified, formatStyle will return 'narrow' if format is 'elapsed' or 'micro', 'short' if the format is 'relative' or 'datetime', otherwise it will be 'long'.